examples/MGI/src/main/kotlin/org/stellar/example/Deposit.kt

```diff
@@ -0,0 +1,42 @@
+package org.stellar.example
+
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.delay
+import org.stellar.walletsdk.anchor.TransactionStatus
+
+object Deposit {
+  @JvmStatic
+  suspend fun main() {
+    // Authorizing
+    val token = anchor.auth().authenticate(myAccount, Signer, clientDomain = clientDomain)
```

Comment:
Where is `Signer` defined?

Reply:
Or is this kotlin's way of passing the default signer?

Reply:
It's defined [here](https://github.com/stellar/kotlin-wallet-sdk/blob/36bb82fffaab2064af08216c532a2363434e315d/examples/MGI/src/main/kotlin/org/stellar/example/Withdrawal.kt#L63), I will move it into it's own file